event/doc.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+// Package event is a library for monitoring events from the MongoDB Go
+// driver. Monitors can be set for commands sent to the MongoDB server and/or
+// changes on the MongoDB server.
+//
+// Monitoring commands requires setting a CommandMonitor object on the
+// mongo.Client. A CommandMonitor can be set to monitor started, succeeded,
+// and/or failed events. For example, the following code collects the names of
+// started events:
+//
+//    var commandStarted []string
```

Comment:
This sort of simple example is fine for the `doc.go` page, but we should also have at least one more in-depth example that registers all three listeners. Maybe something that uses the started event to store `RequestID`, uses that to match with the finding finished event, and then logs the request and response in the success/failure handler?

It'd probably make sense to do this as an example on the `CommandMonitor` type.

Reply:
done